tools/monorepo-utils/src/code-freeze/commands/changelog/index.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+import { execSync } from 'child_process';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../core/logger';
+import { cloneAuthenticatedRepo } from '../../../core/git';
+import { updateTrunkChangelog, updateReleaseBranchChangelogs } from './lib';
+import { Options } from './types';
+
+export const changelogCommand = new Command( 'changelog' )
+	.description( 'Create a new release branch' )
+	.option(
+		'-o --owner <owner>',
+		'Repository owner. Default: woocommerce',
+		'woocommerce'
+	)
+	.option(
+		'-n --name <name>',
+		'Repository name. Default: woocommerce',
+		'woocommerce'
+	)
+	.option(
+		'-d --devRepoPath <devRepoPath>',
```

Comment:
Food for thought, but, it's probably better for us to have hyphenated options instead of camelCase ones.